src/styles/daigest.scss

```
@@ -0,0 +1,107 @@
+.daigest {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  height: 459px;
+  max-width: $max-monitor;
+  padding: 160px 55px;
+  background-color: #cd4d31;
+
+  &__title {
+    margin-bottom: 41px;
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 120%;
+
+    color: $white-text;
+  }
+
+  &__text {
+    width: 570px;
+
+    @include Raleway-small-font;
+
+    font-weight: 200;
+    line-height: 150%;
+    color: $white-text;
+
+  }
+
+  &__box {
+    display: flex;
+  }
+
+  &__email {
+    margin-top: 20px;
+    margin-left: 145px;
+    width: 290px;
+
+    border: none;
+    border-bottom: 1px solid #f1f5f4;
+    outline: none;
+    background-color: #cd4d31;
+    color: white;
+
+    &::placeholder {
+      color: $white-text;
+    }
+  }
+
+  @media screen and (max-width: 1050px) {
+    & {
+      height: 532px;
+      padding-top: 110px;
+      padding-bottom: 120px;
+      padding-left: 99px;
+      padding-right: 99px;
```

```suggestion
      padding: 110px 99px 120px;
```
И да, откуда такие магические цифры? 99?

---

исправил запись, за цифры не помню, подгонял видать переходной вариант под что-то выше чтоб красиво было :)